src/main/scala/scalismo/package.scala

```diff
@@ -38,6 +38,10 @@ package object scalismo {
       gc.SetScheduleTime(60, TimeUnit.SECONDS)
       gc.Start()
 
+      // we need to stop the garbage collection when the JVM shutdowns.
+      // Otherwise it prevents the JVM from shutting down correctly.
+      Runtime.getRuntime().addShutdownHook(new Thread { gc.Stop() })
```

Comment:
@clangguth The problem seems to be that the garbage collector does not run in a daemon thread and hence is not terminated at the end. Maybe something like this would work? What do you think?
```scala
      val gcThread = new Thread {
        override def run() {
          while (true) {
            Thread.sleep(60 * 1000) // wait for 60 seconds
            val gc = vtkObjectBase.JAVA_OBJECT_MANAGER.gc(false)
          }
        }
      }
      gcThread.setDaemon(true)
      gcThread.start()
```